commons/src/main/java/gov/cms/qpp/conversion/model/validation/MeasureConfigs.java

```diff
@@ -49,6 +49,12 @@ private MeasureConfigs() {
 		//empty and private constructor because this is a singleton
 	}
 
+	/**
+	 * Method to ensure MeasureConfigs is initialized
+	 */
+	public static void init() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)